mulighetsrommet-arena-adapter/src/test/kotlin/no/nav/mulighetsrommet/arena/adapter/ArenaAdapterTestConfig.kt

```diff
@@ -0,0 +1,100 @@
+package no.nav.mulighetsrommet.arena.adapter.no.nav.mulighetsrommet.arena.adapter
+
+import io.ktor.server.testing.*
+import no.nav.common.kafka.util.KafkaPropertiesBuilder
+import no.nav.common.token_client.builder.AzureAdTokenClientBuilder
+import no.nav.mulighetsrommet.arena.adapter.AppConfig
+import no.nav.mulighetsrommet.arena.adapter.AuthConfig
+import no.nav.mulighetsrommet.arena.adapter.AuthProvider
+import no.nav.mulighetsrommet.arena.adapter.KafkaConfig
+import no.nav.mulighetsrommet.arena.adapter.ServiceClientConfig
+import no.nav.mulighetsrommet.arena.adapter.ServiceConfig
+import no.nav.mulighetsrommet.arena.adapter.TaskConfig
+import no.nav.mulighetsrommet.arena.adapter.TopicsConfig
+import no.nav.mulighetsrommet.arena.adapter.configure
+import no.nav.mulighetsrommet.arena.adapter.createRSAKey
+import no.nav.mulighetsrommet.arena.adapter.services.ArenaEventService
+import no.nav.mulighetsrommet.arena.adapter.tasks.RetryFailedEvents
+import no.nav.mulighetsrommet.database.kotest.extensions.createArenaAdapterDatabaseTestSchema
+import no.nav.security.mock.oauth2.MockOAuth2Server
+import org.apache.kafka.common.serialization.ByteArrayDeserializer
+
+fun <R> withArenaAdapterApp(
+    oauth: MockOAuth2Server = MockOAuth2Server(),
+    config: AppConfig = createTestApplicationConfig(oauth),
+    test: suspend ApplicationTestBuilder.() -> R
+) {
+    val tokenClient = AzureAdTokenClientBuilder.builder()
```

Comment:
Denne benyttes vel ikke av noen av testene som du har implementert, men kan benytte de samme verdiene som vi gjør i `DevApplication.kt` slik at oppsettet stemmer med MockOauth2Server (som du også har lagt til).

Egentlig så hadde det vært kjekt om vi hadde klart å få ned mengden boilerplate når det kommer til config-oppsettet for testapplikasjonen, vi kan se på det i en egen huddle kanskje :)